coalib/results/SourceRange.py

```diff
@@ -126,6 +126,27 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Default representation::
```

Comment:
Yeah sure, first I'll let the representations as it is right now, and then provide three doctests corresponding to each representation?